iot-hub/Samples/service/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs

```diff
@@ -90,24 +99,40 @@ private static void CreateMetricsAndTargetCondition(Configuration configuration)
 
         private async Task DeleteConfigurationAsync(string configurationId)
         {
-            await _registryManager.RemoveConfigurationAsync(configurationId);
+            try
+            {
+                await _registryManager.RemoveConfigurationAsync(configurationId);

```

Comment:
This request won't hit job throttling limits since it is not a job request, so you can skip the try/catch here